src/services/rounds.service.js

```diff
@@ -2,14 +2,31 @@ import mongoose from 'mongoose';
 import Round from '../models/round';
 import uuid from 'uuid'
 class RoundService {
-  getRounds(){
+  getRounds() {
     return Round.find({})
   }
+  
   createRound(query) {
-    return Round.create({ 
+    return Round.create({
       ...query
     });
   }
+
+  byEventId(event_id, based_on_event_id) {
+    return Round.find({
```

Comment:
can you throw error in case you don't get `event_id` and `based_on_event_id`?

Reply:
yes